tendrl/node_agent/message/handler.py

```diff
@@ -57,7 +58,12 @@ def bind_unix_listener(self):
             self.sock.setblocking(0)
             self.sock.bind(socket_path)
             self.sock.listen(50)
-        except (TypeError, BlockingIOError, socket_error, ValueError) as ex:
+            uid = pwd.getpwnam("tendrl-user").pw_uid
+            os.chown(socket_path, uid, -1)
```

Comment:
The socket permissions should be `770`, owned by `root` and group ownership set to `tendrl`.